src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverted = {};
+
+  for (const prop in robot) {
+    if (inverted[robot[prop]] !== undefined) {
+      return null;
```

Comment:
do you need to return null here? Seems `break` or `continue` might come in handy here

Reply:
disregard this comment, i found out it is required in task description